### PR TITLE
TPC SpaceCharge: make SpaceCharge object writable to file

### PR DESCRIPTION
**Pull request by matthias-kleiner**
-TPCFastTransform can also be stored/loaded from a root file with the TPCSlowSpaceChargeCorrection enabled

### CODE REVIEW

File: Detectors/TPC/spacecharge/include/TPCSpaceCharge/SpaceCharge.h [hunk 1]
```diff
@@ -910,93 +910,56 @@ class SpaceCharge
...
+  inline static int sSimpsonNIteratives{3};                                                               ///<! number of iterations which are performed in the iterative simpson calculation of distortions/corrections
+  inline static int sSteps{1};                                                                            ///<! during the calculation of the corrections/distortions it is assumed that the electron drifts on a line from deltaZ = z0 -> z1. The value sets the deltaZ width: 1: deltaZ=zBin/1, 5: deltaZ=zBin/5
+  inline static GlobalDistType sGlobalDistType{GlobalDistType::Fast};                                     ///<! setting for global distortions: 0: standard method,      1: interpolation of global corrections
+  inline static GlobalDistCorrMethod sGlobalDistCorrCalcMethod{GlobalDistCorrMethod::LocalDistCorr};      ///<! setting for  global distortions/corrections: 0: using electric field, 1: using local dis/corr interpolator
+  inline static SCDistortionType sSCDistortionType{SCDistortionType::SCDistortionsConstant};              ///<! Type of space-charge distortions
```
Comment (wiechula) [inline Detectors/TPC/spacecharge/include/TPCSpaceCharge/SpaceCharge.h:920]: if you want to forbid streaming, I think it should be `//!< ...`. But I'm not sure this is at all necessary for static variables.

For curiosity, why did you remove the comments below?

Comment (matthias-kleiner) [reply Detectors/TPC/spacecharge/include/TPCSpaceCharge/SpaceCharge.h:920]: Indeed, I think for static variables it doesnt matter, but `///<!` also works. see https://root.cern/manual/io/:
> To accommodate doxygen-style documentation, this annotation can also be written as ///<!.

> For curiosity, why did you remove the comments below?

I didnt remove them, they are just at the far right now, since it didnt work to exclude them from being streamed when they have a line break like 
```
  TriCubic mInterpolatorPotential[FNSIDES]{
    {mPotential[Side::A], mGrid3D[Side::A]},
    {mPotential[Side::C], mGrid3D[Side::C]}}; ///<! interpolator for the potenial
```
so it is now `  TriCubic mInterpolatorPotential[FNSIDES]{{mPotential[Side::A], mGrid3D[Side::A]},{mPotential[Side::C],mGrid3D[Side::C]}};                                                                                                                                                                                                                                                                                                                                                                           ///<! interpolator for the potenial`

**Review by wiechula — APPROVED** [review]
Thanks, I overlooked that!
